[ADD] Appraisals: Moving goals to new doc (#9429)
odoo/documentation: content/applications/hr/appraisals/goals.rst

=====
Goals
=====

> The ability to set goals for employees to work toward is an important feature of Odoo *Appraisals*.
> Goals are typically set during an appraisal, so the employee knows what to work toward before their
> next appraisal.

Reviewer: ```suggestion
Set goals in Odoo *Appraisals*, so employees know what to work toward before their next review.
```

To view all goals, navigate to :menuselection:`Appraisals application --> Goals`. This presents all
the goals for every employee, in a default Kanban view.

Each goal card contains the following information:

- :guilabel:`Goal`: the name of the goal.
- :guilabel:`Employee`: the employee the goal is assigned to.

> - :guilabel:`Activities`: displays the corresponding :doc:`activity icon

Reviewer: i was confused by this, as it was not the word "Activities" shown in the card, but the activities icon.

Maybe we can change this to the clock icon instead?
```suggestion
- :icon:`fa-clock-o` (:guilabel:`clock`) icon: displays the corresponding :doc:`activity icon
```

Author: Sure- it's hard to define since it can appear as several icon- whatever is scheduled is what appears (a group of people, an envelope, etc). But I'll use the clock here!

  <../../essentials/activities>` for the record.
- :guilabel:`Deadline`: the due date for the goal.

> - :guilabel:`Progress`: the percentage of competency set for the goal.

Reviewer: Just wondering: is the progress always in 25% increments?

If so, maybe we can be a little bit more specific about the UI, such as:
```suggestion
- :guilabel:`Progress`: 0%, 25%, 50%, 75%, or 100% to represent progress towards the goal.
```

Author: Yes, that is how they set the progress, so you can't change it to like, 30% for example.

- :guilabel:`Employee Icon`: the profile icon of the employee the goal is assigned to.

If a goal is completed, a :guilabel:`Done` banner appears in the top-right corner of the goal card.

.. image:: goals/goals.png
   :align: center
   :alt: The goals Kanban view, with nine goal cards.

.. note::
   Every individual goal requires its own record for each employee. If multiple employees have the
   same goal, a goal card for each employee appears on the list.

   For example, if both Bob and Sara have the same goal of `Typing`, two cards appear in the Kanban
   view: one `Typing` goal for Bob, and another `Typing` goal for Sara.

New goal
========

To create a new goal, navigate to :menuselection:`Appraisals application --> Goals`, and click
:guilabel:`New` top-left corner to open a blank goal form.

Enter the information on the goal form. The information requested is all the same information that

> appears on the goal card  in the Kanban view, with the addition of a :guilabel:`Tags` field and a

Reviewer: > Enter the information on the goal form.

Can be a little bit more specific. How about something like:

"Input the Goal, Employee, Progress, and Deadline information as discussed in the [goal card section] (ref link to the earlier part" ?

this document.

The information requested is all the same information that appears on the goal card in the Kanban
view, with the addition of a :guilabel:`Tags` field and a :guilabel:`Description` tab.

.. tip::
   The *Appraisals* application does **not** have any pre-configured tags, so all tags need to be
   added. To add a tag, enter the name of the tag on the line, then click :guilabel:`Create
   "(tag)"`. Repeat this for all tags that need to be added.

The current user populates the :guilabel:`Employee` field, by default, and the :guilabel:`Manager`
field populates with the manager set on the employee profile.

Make any necessary changes to the form, and add any notes that might be useful to clarify the goal
in the :guilabel:`Description` tab.

.. image:: goals/new-goal.png
   :align: center
   :alt: A goal form filled out for a Python skill, set to 50% proficiency.

Completed goals
===============

When a goal has been met, it is important to update the record. A goal can be marked as `Done` in
one of two ways: from the main :guilabel:`Goals` dashboard, or from the individual goal card.

> To mark a goal as `Done` from the main :guilabel:`Goals` dashboard, click on the
> :icon:`fa-ellipsis-v` :guilabel:`(Dropdown menu)` icon in the top-right of a goal card.
> 
> .. important::
>    The :icon:`fa-ellipsis-v` :guilabel:`(Dropdown menu)` icon **only** appears when the mouse
>    hovers over the top-right corner of a goal card.

Reviewer 2: Update Icon label :) please use the icon's name
```suggestion
To mark a goal as `Done` from the main :guilabel:`Goals` dashboard, click on the
:icon:`fa-ellipsis-v` :guilabel:`(vertical ellipsis)` icon in the top-right of a goal card.

.. note::
   The :icon:`fa-ellipsis-v` :guilabel:`(vertical ellipsis)` icon **only** appears when the mouse
   hovers over the top-right corner of a goal card.
```

Author: Ahh, ok- because you visually see the words "Dropdown menu" when you rollover the icon, I assumed it was OK for me to use the name.
